paywall/src/components/lock/UnlockFlag.js

```diff
@@ -10,7 +10,9 @@ export function LockedFlag() {
   return (
     <Colophon>
       <RoundedLogo size="28px" />
-      <p>Powered by Unlock</p>
+      <p>
+        Powered by&nbsp;<a href="https://paywall.unlock-protocol.com">Unlock</a>
```

Comment:
I think we should use `/` here rather?

Reply:
Oh you mean linking to `/` instead of a full url, that's not a bad idea.

Reply:
exactly ;)